### ISSUE TITLE
Broken "First Steps" documentation

### ISSUE THREAD

**Issue by eigenhombre** [pallet/pallet.github.com#22]
http://palletops.com/doc/first-steps/ is broken for me where it says, 

```
To find the value for other clouds, you can list the supported providers with:
bash$ lein pallet providers
```

my output is at https://gist.github.com/3076544.

Thanks,
John

**Comment by hugoduncan** [comment]
John,

Could you add the lein version you are using, and the generated project.clj file.

Thanks, 
Hugo

**Comment by eigenhombre** [comment]
$ lein version
Leiningen 1.7.1 on Java 1.6.0_33 Java HotSpot(TM) 64-Bit Server VM
$ cat project.clj 
(defproject quickstart "0.1.0-SNAPSHOT"
  :description "FIXME Pallet project for quickstart"
  :dependencies [[org.clojure/clojure "1.3.0"]
                 [org.cloudhoist/pallet "0.7.1"]
                 [org.cloudhoist/pallet-jclouds "1.3.0-beta.1"]
                 ;; To get started we include all jclouds compute providers.
                 ;; You may wish to replace this with the specific jclouds
                 ;; providers you use, to reduce dependency sizes.
                 [org.jclouds/jclouds-allblobstore "1.3.2"]
                 [org.jclouds/jclouds-allcompute "1.3.2"]
                 [org.jclouds.driver/jclouds-slf4j "1.3.2"]
                 [org.slf4j/slf4j-api "1.6.1"]
                 [ch.qos.logback/logback-core "1.0.0"]
                 [ch.qos.logback/logback-classic "1.0.0"]]
  :dev-dependencies [[org.cloudhoist/pallet "0.7.1" :type "test-jar"]
                     [org.cloudhoist/pallet-lein "0.5.0"]]
  :profiles {:dev {:dependencies [[org.cloudhoist/pallet-lein "0.5.0"]]}}
  :local-repo-classpath true
  :repositories
  {"sonatype-snapshots" "https://oss.sonatype.org/content/repositories/snapshots"
   "sonatype" "https://oss.sonatype.org/content/repositories/releases/"})
$

**Comment by eigenhombre** [comment]
Sorry, that's hard to read.  https://gist.github.com/3077676 is easier.

**Comment by hugoduncan** [comment]
The problem is in the generated project.clj (the plugin should be under :plugins rather than under :dependencies). This should be fixed in pallet/lein-template 0.2.4 - looks like I forgot to update the first-steps :(

Thanks for the report. Reopen if this doesn't fix things.

**Comment by eigenhombre** [comment]
Thanks!
